documentation/site/content/index.html

```diff
@@ -51,6 +51,11 @@ <h2 class="govuk-heading-m">Product documentation</h2>
   <li><a class="govuk-link--no-visited-state" href="/product/api-versioning/">API Change Management Guidelines</a></li>
 </ul>
 
+<h2 class="govuk-heading-m">Content editors</h2>
+
+<ul class="govuk-list">
+  <li><a class="govuk-link--no-visited-state" href="/content-editors/api-guidance-pages/">Lead Provider API Guidance Pages</a></li>
+</ul>
```

Comment:
I don't think we need a singular section for this, I can't imagine we're going to add many more content editor articles. Is it ok if we put below section for useful resources please?

Reply:
done 👍

Reply:
thank you